src/Pelagos/Util/DOIutil.php

```diff
@@ -71,9 +72,6 @@ public function __construct()
      * @param string $status          Status of the DOI, by default is reserved.
      * @param string $resourcetype    Type for DOI Request, by default Dataset.
      *
-     * @throws HttpClientErrorException When there was an 4xx error negotiating with EZID.
```

Comment:
If you don't handle (try) the calling function, then this function can throw exceptions.

Reply:
I'm stuck with having to catch a 400 (or 404) specifically. The rest I want to re-throw.